Formula/fabric-installer.rb

```diff
@@ -0,0 +1,20 @@
+class FabricInstaller < Formula
+  desc "Installer for Fabric for the vanilla launcher"
+  homepage "https://fabricmc.net/"
+  url "https://maven.fabricmc.net/net/fabricmc/fabric-installer/0.7.3/fabric-installer-0.7.3.jar"
+  sha256 "a3902483785e6d80e794c81ba372c91d244684827cce176c1a9d605aec416fa7"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    libexec.install "fabric-installer-#{version}.jar"
+    bin.write_jar_script libexec/"fabric-installer-#{version}.jar", "fabric-installer"
+  end
+
+  test do
+    system "#{bin}/fabric-installer", "server"
```

Comment:
Can we check some output or interaction here?

Reply:
This command generates a `fabric-server-launch.jar` file. Should this test check for that instead?

Reply:
That sounds like a good idea

Reply:
Fixed in 616eee4c70bf0be0b285d2b2ed56a3af705e28e6